tests/aws/services/lambda_/test_lambda_api.validation.json

```diff
@@ -57,11 +57,26 @@
     "last_validated_date": "2024-04-10T08:59:14+00:00"
   },
   "tests/aws/services/lambda_/test_lambda_api.py::TestLambdaFunction::test_function_arns": {
-    "last_validated_date": "2024-04-10T09:00:24+00:00"
+    "last_validated_date": "2024-08-13T13:09:05+00:00"
   },
   "tests/aws/services/lambda_/test_lambda_api.py::TestLambdaFunction::test_function_lifecycle": {
     "last_validated_date": "2024-04-10T08:59:22+00:00"
   },
+  "tests/aws/services/lambda_/test_lambda_api.py::TestLambdaFunction::test_function_name_and_qualifier_validation": {
+    "last_validated_date": "2024-08-14T15:25:00+00:00"
+  },
```

Comment:
Would be nice to clean this up before merging, I think it is a leftover :)

Reply:
Removed